rules/prefer-string-starts-ends-with.js

```diff
@@ -4,12 +4,20 @@ const getDocumentationUrl = require('./utils/get-documentation-url');
 const methodSelector = require('./utils/method-selector');
 const quoteString = require('./utils/quote-string');
 const shouldAddParenthesesToMemberExpressionObject = require('./utils/should-add-parentheses-to-member-expression-object');
+const shouldAddParenthesesToLogicalExpressionChild = require('./utils/should-add-parentheses-to-logical-expression-child');
+const {getParenthesizedText} = require('./utils/parentheses');
 
 const MESSAGE_STARTS_WITH = 'prefer-starts-with';
 const MESSAGE_ENDS_WITH = 'prefer-ends-with';
+const SUGGEST_STRING_CAST = 'suggest-string-cast';
```

Comment:
Let's give these two constants a  better name, so we can use  

```js
suggest: [A, B, C].map(type => ({messageId: type, fix: fix(node, type}))
``

And also use these constants to compare `type` in `fix` function.

Reply:
Cool idea, updated.

Reply:
Not the thing in my mind, let me commit my idea.

Reply:
Check https://github.com/sindresorhus/eslint-plugin-unicorn/pull/1277/commits/cb5e41dbcede4f2a0f4a49cb7aedcce6cef3cdff, better?

Reply:
That works, I just pushed a commit to finish that refactoring.